app/views/content/returning-to-teaching.md

```diff
@@ -130,25 +138,26 @@ $q-helen-returner$
+If you’re returning to teaching, you might want to re-familiarise yourself with life in a classroom. Here are lots of ways that you can do it.
 
-If you’re unsure about taking on a teacher role at this time, [consider applying to tutor or mentor through the National Tutoring Programme](https://nationaltutoring.org.uk/) and help provide support to disadvantaged pupils.
+#### Reading teachers' stories and tips
 
-## If you've been out of teaching for a while
+* browse [inspirational stories](https://getintoteaching.education.gov.uk/blog/tag/returning-to-teaching) from Stephen, Farooq and Maria who returned to the classroom successfully after a break
+* read up on [teaching interview tips](https://www.teachertoolkit.co.uk/2019/04/04/job-interviews) and get [pointers on remote interviews](https://www.tes.com/news/coronavirus-10-tips-acing-remote-job-interview)
 
-There’s lots you can do to re-familiarise yourself with teaching. For example, you can:  
+#### Catching up on changes
 
-* attend an [online return to teaching event](/event-categories/online-q-as)
-* read [blog posts about returning to teaching](/blog/tag/returning-to-teaching)
-* read [Jill's journey](/my-story-into-teaching/returners/getting-back-into-the-classroom) to become a teacher again
 * review the latest [Teachers’ Standards](https://www.gov.uk/government/publications/teachers-standards)
 * catch up on changes to the [national curriculum and qualifications](https://www.gov.uk/topic/schools-colleges-childrens-services/curriculum-qualifications)
 * [read through current behaviour management guidelines](https://www.gov.uk/government/publications/behaviour-and-discipline-in-schools)
+* learn about how DfE have improved the [National Professional Qualifications (NPQs)](https://www.gov.uk/government/publications/national-professional-qualifications-npqs-reforms/national-professional-qualifications-npqs-reforms) and if you may be eligible for a fully funded scholarship 
+
+#### Things to do
+
+* attend an [online return to teaching event](/event-categories/online-q-as)
 * join the [Chartered College of Teaching](https://chartered.college/), the
   professional body for teachers
 * join a [teaching union](https://www.tes.com/jobs/careers-advice/pay-and-conditions/which-teachers-union)
-* read up on [teaching interview tips](https://www.teachertoolkit.co.uk/2019/04/04/job-interviews)
-  and [get pointers on remote interviews](https://www.tes.com/news/coronavirus-10-tips-acing-remote-job-interview)
-* ensure your [pension is up to date](https://www.teacherspensions.co.uk/members/working-life/life-events/moving-abroad.aspx)
+* ensure your [pension](https://www.teacherspensions.co.uk/members/working-life/life-events/moving-abroad.aspx), one of your key benefits, is up to date
```

Comment:
This is a strange page for this to go to (moving abroad) - should it go to the pensions home page instead?

(I appreciate it goes to the moving abroad page in production!)

Reply:
Yes, I thought this when I first clicked on the link! Then I scrolled down the page and saw that there is a part of the page specifically about pensions if you're returning to teaching so was planning on asking how to do a jump link, but then forgot when I've put the changes into Github. @peteryates would you be able to help with this? :)

Reply:
@EllieNodder True (and I didn't notice that!) **but** that is still talking about overseas pensions/international returners, which is not the audience this page is aimed at?

I don't think a jump link is the answer - I think it needs to go to a different page? Worth following up with the returners team.

Reply:
Good point - will speak to Sharon